include/graphblas/base/config.hpp

```diff
@@ -33,6 +33,7 @@
 
 #include <assert.h>
 #include <unistd.h> //sysconf
+#include <cstdint>
```

Comment:
maybe add `// for size_t`, if indeed it is added because of that?

(also elsewhere)